Sources/StreamVideoSwiftUI/CallView/VideoParticipantsView.swift

```diff
@@ -452,8 +455,10 @@ public struct ParticipantInfoView: View {
             SoundIndicator(participant: participant)
                 .frame(maxHeight: maxHeight)
         }
-        .padding(.all, 2)
-        .padding(.horizontal, 4)
+        .padding(.leading, paddingsConfig.leading)
```

Comment:
Does it make sense to create simple modifier that applies the paddingsConfig directly so we don't have to add 4 lines of padding configuration per view + make sure that always the right value is being applied on the right side?